app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
```

Comment:
- [ ] [OPTIONAL] kindly try to keep to the given wireframe, your registration page and login page does not look like the given wireframe
your design :point_down: 
![Screenshot from 2023-09-09 01-41-15](https://github.com/benja27/my-first-blog-with-rails/assets/66097371/91d8f5a9-1004-491a-befa-0f45dc9857ea)
given design :point_down: 
![Screenshot from 2023-09-06 19-26-16](https://github.com/benja27/my-first-blog-with-rails/assets/66097371/0abd221e-37b0-4ec7-86cd-92d51229fa99)